src/shared/components/modal.jsx

```diff
@@ -22,7 +22,6 @@ const Backdrop = styled("div")`
   bottom: 0;
   left: 0;
   right: 0;
-  z-index: 1300;
```

Comment:
Couple things here:

(1) I would keep the `z-index` declaration in the styled component itself. You could easily add it like this:

```
const Backdrop = styled('div')`
...
z-index: ${props => props.zIndex || 1300};
...
`
```
(2) it might be a better and more tractable thing to not force me to understand the inner workings of the z-index in our modals by just having a keyword that says it needs to be on top of the other z-indices. Perhaps there is just a `nested` (true/false) which make the `z-index` 1400, otherwise the `z-index` is 1300. I think we probably won't ever have more than the 3 levels of z-index we have here so I think this should work nicely in theory.

Reply:
I like this. Good suggestion.

Reply:
Also, keeping the `z-index` declaration in the style component is going to require moving `Backdrop` into the constructor where `props` is in scope. Just FYI if you see some big whitespace changes.

Reply:
oh, plz add anything relating to z-indexes as constants in "src/editor/style/z-index-styles". we do some funky things with the layout manager, so i want to make sure that we can have an overview of the z-index twiddles in one place.

(@hamilton if you want to merge these constants into `THEME` that is fine with me, but all the Z stuff should be in one place; either this file or THEME.layers or something)